demos/serverless/deploy.js

```diff
@@ -36,6 +37,7 @@ let mediaCaptureRegions = [
 function usage() {
   console.log(`Usage: deploy.sh [-r region] [-b bucket] [-s stack] [-a application] [-e]`);
   console.log(`  -r, --region                         Target region, default '${region}'`);
+  console.log(`  -u, --use-regional                   Flag to switch between chime and regional client, default '${useRegional}'`);
```

Comment:
Naming: Please provide context, eg `--regional-sdk` or `--regional-aws-sdk`

Reply:
Similarly, refrain from "global" and "regional" terms

Reply:
updated